app/mailers/course_update_email_mailer.rb

```diff
@@ -0,0 +1,29 @@
+class CourseUpdateEmailMailer < GovukNotifyRails::Mailer
+  def send_course_update_email(course, attribute_changed)
```

Comment:
This should just be `course_update_email`. This method just creates it, it will be sent with `course_update_email(...).deliver`